README.md

```diff
@@ -85,6 +85,24 @@ This class will provide us with the timer for the scheduler where it will provid
 ##Configurations.java
 The configuration needed across many other classes.
 
+##Timing.java
+A static Timing class to measure the time it takes to finish servicing all of the events in the input file. It includes methods to start the timer, record when we�re done servicing an event, and then get timing info when the whole input file is complete. It is calculated by taking average time for one request by taking the total time and dividing it by the total number of requests because requests do not finish sequentially in our real time system.
+
+##CustomRenderer.java
+A class set so that it can force the background of the ElevatorGUI to be colored in red, white, or black.
```

Comment:
I think you need to add a space after the ## so that it looks right when it converts to MD @yuvrajzala26 (this applies to the whole file